docker-compose/test.yml

```diff
@@ -1,5 +1,5 @@
 kuzzle:
-  image: kuzzleio/kuzzle
+  image: kuzzleio/dev
```

Comment:
I think it's still `kuzzleio/kuzzle` https://hub.docker.com/r/kuzzleio/kuzzle/~/dockerfile/